asapdiscovery-data/asapdiscovery/data/openeye.py

```diff
@@ -293,15 +311,39 @@ def load_openeye_sdfs(sdf_fn):
         oechem.OEThrow.Fatal(f"Unable to open {sdf_fn}")
 
 
-def load_openeye_design_unit(du_filename: str) -> oechem.OEDesignUnit:
+def load_openeye_design_unit(du_fn: Union[str, Path]) -> oechem.OEDesignUnit:
+    """
+    Load an OpenEye design unit from a file
+
+    Parameters
+    ----------
+    du_fn : Union[str, Path]
+        The path of the DesignUnit file to read.
+
+    Returns
+    -------
+    oechem.OEDesignUnit
+        OpenEye DesignUnit
+
+    Raises
+    ------
+    FileNotFoundError
+        If the specified file does not exist.
+    oechem.OEError
+        If the CIF file cannot be opened.
+
+
+    """
+    if not Path(du_fn).exists():
+        raise FileNotFoundError(f"{du_fn} does not exist!")
     du = oechem.OEDesignUnit()
-    retcode = oechem.OEReadDesignUnit(du_filename, du)
+    retcode = oechem.OEReadDesignUnit(str(du_fn), du)
     if not retcode:
-        raise RuntimeError(f"Unable to read design unit from {du_filename}")
+        oechem.OEThrow.Fatal(f"Unable to open {du_fn}")
```

Comment:
This is maybe its own issue / PR, but have you confirmed that all the scripts / code that uses this can handle the openeye error logging correctly? I guess this is #327

Reply:
This is only used in single target workflow so far, but its a good point. My instinct is that if you don't meddle with the OE streams (and I think we found a solution to do that anyway) this should be OK.